lib/src/main/java/graphql/nadel/engine/NadelExecutionContext.kt

```diff
@@ -25,10 +25,6 @@ data class NadelExecutionContext internal constructor(
 ) {
     private val serviceContexts = ConcurrentHashMap<String, CompletableFuture<Any?>>()
 
-    @Deprecated("Use incrementalSupport instead", ReplaceWith("incrementalResultSupport"))
-    internal val deferSupport: NadelIncrementalResultSupport
-        get() = incrementalResultSupport
-
```

Comment:
Pretty sure I was the only one using this so we can remove the deprecated field now